src/connectors/pagination/connectPagination.js

```diff
@@ -100,10 +100,10 @@ export default function connectPagination(renderFn, unmountFn = noop) {
     return {
       $$type: 'ais.pagination',
 
-      getConfiguration() {
-        return {
+      getConfiguration(state) {
+        return state.setQueryParameters({
           page: 0,
```

Comment:
Should this take into account the previous `page` state now?

Reply:
Yes, it should